### DISCUSSION TITLE
Link the page

### DISCUSSION THREAD

**Discussion by rajeetdon** [conroy-tafesa/T24S2T1EXTG03#3] [General]
How do you link the mortgage calculator page with the main page ?

**Comment by conroy-tafesa** [comment]
For the 'Exit' button on your feature page, this code will take it back to the Main Menu:
```
// Return to the main menu
private void exitButton_Click(object sender, RoutedEventArgs e)
{
	this.Frame.Navigate(typeof(MainMenu));
}
```
Likewise, on the Main Menu page, I have this, though you'll need to adapt it for your page:
```
private void currencyConverter_Click(object sender, RoutedEventArgs e)
{
	this.Frame.Navigate(typeof(CurrencyCalc));
}
```

**Reply by rajeetdon** [reply]
Thank you. Its work
